platform-sdk/build.gradle.kts

```diff
@@ -15,32 +15,33 @@
  */
 
 plugins {
-  id("com.swirlds.platform.aggregate-reports")
-  id("com.swirlds.platform.spotless-conventions")
-  id("com.swirlds.platform.spotless-kotlin-conventions")
+    id("com.swirlds.platform.aggregate-reports")
```

Comment:
This will be addressed in the follow up. Started here: #7875

My proposal is:
- For now: get rid of the `com.swirlds.platform` prefix here and use `com.hedera.hashgraph` for all the convention plugins (will complete this transition in follow up PR)
- Later: Rename `com.hedera.hashgraph` to whatever you want that to be in the future